cmd/changelog-parser/testdata/expected_release_output.md

```diff
@@ -10,7 +10,7 @@ All notable changes to this project will be documented in this file.
 - [Upgrade Instructions](#upgrade-instructions)
 - [Changes](#changes)
 
-## Components
+# Components
```

Comment:
I think the header level prior to these changes followed the hierarchical structure of the document with `# Release Notes` as the root and all that under it being a level below. It seems this PR has bumped most of the content headers 1 header level up. I'm curious about the reasoning behind that ?